app/src/main/java/swati4star/createpdf/fragment/ViewFilesFragment.java

```diff
@@ -132,6 +135,46 @@ public View onCreateView(@NonNull LayoutInflater inflater, ViewGroup container,
 
         // Populate data into listView
         populatePdfList();
+        LinearLayout newdir = root.findViewById (R.id.layout_new);
```

Comment:
Change the name of ids to` new_dir`, `move_to_dir`, `move_to_home_dir`, `delete_dir`